pkgs/by-name/bu/budgie-media-player-applet/package.nix

```diff
@@ -34,7 +38,7 @@ stdenv.mkDerivation (finalAttrs: {
 
   postPatch = ''
     substituteInPlace meson.build --replace-fail "/usr" "$out"
-    substituteInPlace meson_post_install.py --replace-fail '"/", "usr"' "\"$out\""
+    substituteInPlace meson_post_install.py --replace-fail MESON_INSTALL_PREFIX out
```

Comment:
Hmmm, I think this substitution actually does nothing? :thinking: 

![](https://github.com/user-attachments/assets/c6977e41-b51e-4305-b0fb-bd5a70970f07)

Reply:
You're totally right. For some reason I thought `MES_INSTALL_PREFIX` would be install prefix of Meson itself 😆